core/src/main/scala/com/databricks/labs/remorph/discovery/SnowflakeTableDefinitions.scala

```diff
@@ -3,6 +3,7 @@ package com.databricks.labs.remorph.discovery
 import com.databricks.labs.remorph.intermediate.{DataType, StructField}
 import com.databricks.labs.remorph.parsers.snowflake.{SnowflakeLexer, SnowflakeParser, SnowflakeTypeBuilder}
 import org.antlr.v4.runtime.{CharStreams, CommonTokenStream}
+import org.apache.spark.sql.types.MetadataBuilder
```

Comment:
don't depend on `org.apache.spark` from this package, create a class inside of `intermediate` package and depend on that one.

Reply:
ok